doc/api/fs.md

```diff
@@ -109,7 +109,12 @@ non-UTF-8 encoded Buffers to `fs` functions will not work as expected.
 added: v0.5.8
 -->
 
-Objects returned from `fs.watch()` are of this type.
+Objects returned from [`fs.watch()`][] are of this type.
+
+The `listener` callback in `fs.watch()` is attached to its returned FSWatcher's
+`change` event.
```

Comment:
Nit - the wording here is a little awkward. How about something like: _The callback provided to `fs.watch()` receives the returned `FSWatch`'s `change` events._